owiddata/generate-owiddata-stats.py

```diff
@@ -4,11 +4,39 @@
 import os
 import pandas as pd
 import geopandas
-import pycountry
+import urllib.request
+from bs4 import BeautifulSoup
 import matplotlib as mpl
 import matplotlib.pyplot as plt
 import numpy as np
 
+def platforms(vaxtype):
+    """The types of vaccines as categoried on trackvaccines.org differs
+    somewhat from the categories we use here.
+    See https://covid19.trackvaccines.org/types-of-vaccines/
+    This function maps their categories (key) onto the section headings
+    used in the manuscript (value)
+    Input: string
+    Output: string"""
+
+    types = {"protein subunit": "subunit",
+             "VLP": "subunit",
+             "plasmid vectored": "DNA",
+             "DNA": "DNA",
+             "RNA": "RNA",
+             "non replicating viral vector": "DNA",
+             "replicating viral vector": "DNA",
+             "inactivated": "whole virus",
+             "live attenuated": "whole virus"
+             }
+
+    # If they add a new category (which seems unlikely), handle & throw error
+    if vaxtype not in types.keys():
+        print("Unknown vaccine platform:", vaxtype)
+        return vaxtype
```

Comment:
I don't think this should ever happen because I think it handles all the major categories of vaccines that currently exist? I just wrote it in as a just-in-case. So I think either would be fine!

Reply:
In that case, I prefer we exit with an error.  That should happen rarely if ever, so we would want to know about the new major vaccine category.